src/styles/card.scss

```diff
@@ -4,12 +4,21 @@
   align-items: center;
   width: 200px;
   padding: 30px 14px 14px;
-  border: 1px solid #f3f3f3;
+  border: 1px solid $cardBorderColor;
   border-radius: 5px;
 
   &__photo {
     width: 160px;
     height: 134px;
+    transition: transform 1s;
+
+    &:hover {
+      box-sizing: content-box;
+      outline: 3px solid $cardBorderColor;
+      margin: -14px 0;
```

Comment:
-14?

Reply:
fix comments before submitting!

I approve this work because comment is not critical